supervisor/bootstrap.py

```
@@ -221,6 +223,13 @@
         )
         config.path_addon_configs.mkdir()
 
+def warning_handler(message, category, filename, lineno, file=None, line=None):
+    """Custom warning handler that logs warnings using the logging module."""
```

Ruff (0.8.2)

First line of docstring should be in imperative mood: "Custom warning handler that logs warnings using the logging module."

(D401)